eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -185,7 +194,7 @@ object Features {
     val mandatory = 6
   }
 
-  case object VariableLengthOnion extends Feature with InitFeature with NodeFeature with InvoiceFeature {
+  case object VariableLengthOnion extends Feature with InitFeature with NodeFeature with Bolt11Feature with Bolt12Feature {
```

Comment:
Actually the specification only seems to allow `basic_mpp` in the Bolt 12 features, `var_onion_optin` and `option_route_blinding` seem to be mandatory but implicit.

I reworked this slightly in https://github.com/ACINQ/eclair/commit/9bf9941b9b3bb3d073baef5d65bd19d063010c4e, I think this way it makes more sense and is easier to work with, what do you think?